server/test/footyPoints.test.js

```diff
@@ -0,0 +1,57 @@
+const mongoose = require("mongoose")
+const createServer = require("./testingTools")
+var alumniSchema = require('../models/alumniSchema');
+const request = require('supertest');
+
+
+const app = createServer();
+
+/**
+ * This runs before all tests in this particular file. I believe
+ * this will needed to be included in every test file to get them
+ * to work right.
+ * I'd bet there's a way to get this to run once per test suite
+ * instead of once per file, but I haven't looked into it yet.
+ */
+beforeAll(async () => {
+    process.env.NODE_ENV = 'test';
+    mongoose.connect("mongodb://localhost:27017/test-test", { useNewUrlParser: true });
+    await request(app).get('/util/seed/');
```

Comment:
Sometimes the /seed API fails due to duplicate key errors. When hitting this manually, it's sufficient to just clear all data and rehit the API.
This should be a easy fix on utilMongoose but we can leave a TODO and move on for now!

Reply:
Which key? The user ID?